llvm/include/llvm/Target/GenericOpcodes.td

```diff
@@ -1500,6 +1500,13 @@ def G_SPLAT_VECTOR: GenericInstruction {
   let hasSideEffects = false;
 }
 
+// Generic masked compress.
+def G_MCOMPRESS: GenericInstruction {
```

Comment:
Should match the IR operation name and spill out the masked_compress

Reply:
done. renamed to `G_MASKED_COMPRESS`